packages/dashboard/e2e/tests/utils/locator.ts

```diff
@@ -0,0 +1,8 @@
+import { Locator } from '@playwright/test';
+
+export type BoundingBox = { x: number; y: number; height: number; width: number };
+export const getBoundingBox = async (locator: Locator): Promise<BoundingBox> => {
+  const bounds = await locator.boundingBox();
+  if (!bounds) throw new Error('could not get bounding box for locator');
+  return bounds;
+};
```

Comment:
Why declare a type and have a return type annotation? TypeScript can infer this return and the `ReturnType` utility type may be used to derive the type from the implementation.